app/components/audio/song-control.tsx

```
@@ -1,61 +1,271 @@
       <div className="p-4 gap-4 flex flex-wrap items-center justify-between">
-        <div className="song-title flex items-center gap-2 w-1/3 md:w-1/6">
-          <Image src={"/assets/placeholder.jpg"} alt="song-playing" width={64} height={64} />
-          <div className="song-title-info">
-            <p className="song-title-text">Song title</p>
-            <p className="song-artist">Artist</p>
+        <div className="song-title flex items-center gap-2 w-fit md:w-1/4">
+          <div className={twMerge(
+            "relative cursor-pointer",
+            (isLoading || isEmpty) && "opacity-50",
+            isLoading && "animate-pulse"
+          )} onClick={handleImageClick}>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)